doc/userguide/rules/smb-keywords.rst

```diff
@@ -0,0 +1,16 @@
+SMB Keywords
+==============
+
+SMB keywords used in both SMB1 and SMB2 protocols.
+
+smb_filename
```

Comment:
smb.filename

Reply:
Done in https://github.com/OISF/suricata/pull/7011/commits/183eb51f809a82b710e403bffbd2779277139bc8